### ISSUE TITLE
Not able to run it

### ISSUE THREAD

**Issue by cja000** [mr-press/DeepChess#1]
Hi,

I'm trying to run DeepChess but I got the following error:

AttributeError: Can't get attribute 'Mul' on <module 'tfdeploy' from '.../tensorflow/lib/python3.5/site-packages/tfdeploy.py'>

Any idea on how to fix it?

Thanks
CJ

**Comment by brianprichardson** [comment]
I got it to work some time ago.  Make sure you have the older tensorflow version 0.10.0 per the readme
Don't recall if python-chess or other older versions might also be required.

**Comment by cja000** [comment]
Thanks for your answer.
The problem was that `tf.mul` changed to `tf.multiply`
So changing, in the `model.pkl` line 93, from "Mul" to "Multiply" it worked.

Now I have another error:
`UnicodeDecodeError: 'ascii' codec can't decode byte 0xd7 in position 1: ordinal not in range(128)`

No clue what "`0xd7`" or "`position 1`" are.

**Comment by brianprichardson** [comment]
Yes, in addition to the mul change there have been many others.  Suggest you create a virtual environment and install the older version of tensorflow there, unless you want to convert the entire project.  There are some tools to try and do that automatically, but you can simply specify the older version more easily.

**Comment by michaeljsong99** [comment]
Do we have to change anything in tfdeploy.py (i.e. writing Ops)? I've been failing on the line:

result = y.eval({x: toEval}) 

in game.py. tf deploy seems to throw an error 

len(indices_or_sections)
TypeError: object of type 'int' has no len()

Thanks!
